diff -up libreoffice-6.2.0/dbaccess/Module_dbaccess.mk.4~ libreoffice-6.2.0/dbaccess/Module_dbaccess.mk
--- libreoffice-6.2.0/dbaccess/Module_dbaccess.mk.4~	2018-12-13 23:33:58.681141153 +0100
+++ libreoffice-6.2.0/dbaccess/Module_dbaccess.mk	2018-12-13 23:36:08.695373909 +0100
@@ -45,9 +45,6 @@ $(eval $(call gb_Module_add_check_target
     CppunitTest_dbaccess_firebird_regression_test \
 ))
 endif
-$(eval $(call gb_Module_add_check_targets,dbaccess,\
-    CppunitTest_dbaccess_firebird_test \
-))
 endif
 
 $(eval $(call gb_Module_add_check_targets,dbaccess,\
