diff -up libreoffice-25.8.1.1/dbaccess/Module_dbaccess.mk.12~ libreoffice-25.8.1.1/dbaccess/Module_dbaccess.mk
--- libreoffice-25.8.1.1/dbaccess/Module_dbaccess.mk.12~	2025-08-26 00:08:27.000000000 +0200
+++ libreoffice-25.8.1.1/dbaccess/Module_dbaccess.mk	2025-09-09 18:47:07.194596563 +0200
@@ -39,10 +39,6 @@ ifeq ($(ENABLE_FIREBIRD_SDBC),TRUE)
 $(eval $(call gb_Module_add_check_targets,dbaccess,\
     $(if $(ENABLE_JAVA),CppunitTest_dbaccess_migration) \
 ))
-
-$(eval $(call gb_Module_add_check_targets,dbaccess,\
-    CppunitTest_dbaccess_firebird_test \
-))
 endif
 
 ifneq ($(filter SCRIPTING,$(BUILD_TYPE)),)
