diff -uraN libreoffice-26.8.0.1/configure.ac omv-libreoffice-26.8.0.1/configure.ac
--- libreoffice-26.8.0.1/configure.ac	2026-07-13 12:47:46.000000000 +0200
+++ omv-libreoffice-26.8.0.1/configure.ac	2026-07-19 21:06:59.051122931 +0200
@@ -6246,14 +6246,7 @@
         bin/verapdf.sh.in \
         instsetoo_native/util/openoffice.lst.in \
         config_host/*.in \
-        sysui/desktop/macosx/Info.plist.in \
-        sysui/desktop/macosx/LaunchConstraint.plist.in \
-        sysui/desktop/macosx/hardened_runtime.xcent.in \
-        sysui/desktop/macosx/lo.xcent.in \
-        .vscode/vs-code-template.code-workspace.in \
         solenv/lockfile/autoconf.h.in \
-        extensions/source/macosx/quicklookpreview/appex/Info.plist.in \
-        extensions/source/macosx/quicklookthumbnail/appex/Info.plist.in \
         ) \
     | (cd CONF-FOR-BUILD && tar xf -)
     cp configure CONF-FOR-BUILD
@@ -16499,13 +16492,6 @@
     if test -n "$WSL_ONLY_AS_HELPER"; then
         sed -i -e 's#/mnt/\([[:alpha:]]\)/#\u\1:/#g' config_host.mk
     fi], [WSL_ONLY_AS_HELPER=$WSL_ONLY_AS_HELPER])
-# also use windows paths in the vs-code workspace file (esp. for the launch configurations that use
-# INSTROOT) and also change the default shell from Cygwin to Git Bash
-AC_CONFIG_FILES([vs-code.code-workspace.template:.vscode/vs-code-template.code-workspace.in], [
-    if test -n "$WSL_ONLY_AS_HELPER"; then
-        sed -i -e 's#/mnt/\([[:alpha:]]\)/#\u\1:/#g' \
-               -e '/defaultProfile/s#Cygwin#Git Bash#' vs-code.code-workspace.template
-    fi], [WSL_ONLY_AS_HELPER=$WSL_ONLY_AS_HELPER])
 
 AC_CONFIG_HEADERS([config_host/config_afdko.h])
 AC_CONFIG_HEADERS([config_host/config_atspi.h])
