diff -up libreoffice-6.2.0/registry/source/regimpl.cxx.omv~ libreoffice-6.2.0/registry/source/regimpl.cxx
--- libreoffice-6.2.0/registry/source/regimpl.cxx.omv~	2018-12-14 01:09:53.878845161 +0100
+++ libreoffice-6.2.0/registry/source/regimpl.cxx	2018-12-14 01:11:46.074887709 +0100
@@ -421,9 +421,7 @@ void dumpType(typereg::Reader const & re
 
 }
 
-#if !HAVE_CPP_INLINE_VARIABLES
 constexpr OUStringLiteral ORegistry::ROOT;
-#endif
 
 ORegistry::ORegistry()
     : m_refCount(1)
diff -up libreoffice-6.2.0/unotools/source/config/optionsdlg.cxx.omv~ libreoffice-6.2.0/unotools/source/config/optionsdlg.cxx
--- libreoffice-6.2.0/unotools/source/config/optionsdlg.cxx.omv~	2018-12-14 03:09:22.732422090 +0100
+++ libreoffice-6.2.0/unotools/source/config/optionsdlg.cxx	2018-12-14 03:09:31.415534086 +0100
@@ -74,9 +74,7 @@ namespace
     class theOptionsDlgOptions_ImplMutex : public rtl::Static<osl::Mutex, theOptionsDlgOptions_ImplMutex>{};
 }
 
-#if !HAVE_CPP_INLINE_VARIABLES
 constexpr OUStringLiteral SvtOptionsDlgOptions_Impl::g_sPathDelimiter;
-#endif
 
 
 ::osl::Mutex & SvtOptionsDlgOptions_Impl::getInitMutex()
diff -up libreoffice-6.2.0/xmloff/source/chart/ColorPropertySet.cxx.omv~ libreoffice-6.2.0/xmloff/source/chart/ColorPropertySet.cxx
--- libreoffice-6.2.0/xmloff/source/chart/ColorPropertySet.cxx.omv~	2018-12-14 14:22:01.779035957 +0100
+++ libreoffice-6.2.0/xmloff/source/chart/ColorPropertySet.cxx	2018-12-14 14:22:08.692137659 +0100
@@ -49,9 +49,7 @@ private:
     Property m_aColorProp;
 };
 
-#if !HAVE_CPP_INLINE_VARIABLES
 constexpr OUStringLiteral lcl_ColorPropertySetInfo::g_aColorPropName;
-#endif
 
 lcl_ColorPropertySetInfo::lcl_ColorPropertySetInfo() :
         m_aColorProp( g_aColorPropName, -1,
diff -up libreoffice-6.2.0/svx/source/form/datanavi.cxx.omv~ libreoffice-6.2.0/svx/source/form/datanavi.cxx
--- libreoffice-6.2.0/svx/source/form/datanavi.cxx.omv~	2018-12-14 20:14:52.556478116 +0100
+++ libreoffice-6.2.0/svx/source/form/datanavi.cxx	2018-12-14 20:15:02.100605441 +0100
@@ -80,14 +80,12 @@ using namespace ::svx;
 namespace svxform
 {
 
-#if !HAVE_CPP_INLINE_VARIABLES
 constexpr OUStringLiteral ReplaceString::m_sDoc_API;
 constexpr OUStringLiteral ReplaceString::m_sInstance_API;
 constexpr OUStringLiteral ReplaceString::m_sNone_API;
 constexpr OUStringLiteral MethodString::m_sPost_API;
 constexpr OUStringLiteral MethodString::m_sPut_API;
 constexpr OUStringLiteral MethodString::m_sGet_API;
-#endif
 
     // properties of instance
     #define PN_INSTANCE_MODEL       "Instance"
diff -up libreoffice-6.2.0/writerfilter/source/dmapper/GraphicImport.cxx.omv~ libreoffice-6.2.0/writerfilter/source/dmapper/GraphicImport.cxx
--- libreoffice-6.2.0/writerfilter/source/dmapper/GraphicImport.cxx.omv~	2018-12-14 20:14:34.274234568 +0100
+++ libreoffice-6.2.0/writerfilter/source/dmapper/GraphicImport.cxx	2018-12-14 20:14:40.408316231 +0100
@@ -383,9 +383,7 @@ public:
     }
 };
 
-#if !HAVE_CPP_INLINE_VARIABLES
 constexpr sal_Int32 GraphicImport_Impl::nFillColor;
-#endif
 
 GraphicImport::GraphicImport(uno::Reference<uno::XComponentContext> const& xComponentContext,
                              uno::Reference<lang::XMultiServiceFactory> const& xTextFactory,
diff -up libreoffice-6.2.0/xmloff/source/chart/ColorPropertySet.cxx.omv~ libreoffice-6.2.0/xmloff/source/chart/ColorPropertySet.cxx
--- libreoffice-6.2.0/xmloff/source/chart/ColorPropertySet.cxx.omv~	2018-12-14 20:15:27.888950088 +0100
+++ libreoffice-6.2.0/xmloff/source/chart/ColorPropertySet.cxx	2018-12-14 20:15:34.049032541 +0100
@@ -81,9 +81,7 @@ namespace xmloff
 namespace chart
 {
 
-#if !HAVE_CPP_INLINE_VARIABLES
 constexpr OUStringLiteral ColorPropertySet::g_aColorPropName;
-#endif
 
 ColorPropertySet::ColorPropertySet( sal_Int32 nColor ) :
         m_nColor( nColor ),
diff -up libreoffice-6.2.0/xmloff/source/chart/SchXMLExport.cxx.omv~ libreoffice-6.2.0/xmloff/source/chart/SchXMLExport.cxx
--- libreoffice-6.2.0/xmloff/source/chart/SchXMLExport.cxx.omv~	2018-12-14 20:15:49.800243597 +0100
+++ libreoffice-6.2.0/xmloff/source/chart/SchXMLExport.cxx	2018-12-14 20:15:53.312290696 +0100
@@ -271,9 +271,7 @@ public:
     OUString maCategoriesRange;
 };
 
-#if !HAVE_CPP_INLINE_VARIABLES
 constexpr OUStringLiteral SchXMLExportHelper_Impl::gsTableName;
-#endif
 
 namespace
 {
diff -up libreoffice-6.2.0/xmloff/source/draw/animexp.cxx.omv~ libreoffice-6.2.0/xmloff/source/draw/animexp.cxx
--- libreoffice-6.2.0/xmloff/source/draw/animexp.cxx.omv~	2018-12-14 20:03:22.670186561 +0100
+++ libreoffice-6.2.0/xmloff/source/draw/animexp.cxx	2018-12-14 20:03:38.254408957 +0100
@@ -241,7 +241,6 @@ public:
     static constexpr OUStringLiteral gsAnimPath = "AnimationPath";
 };
 
-#if !HAVE_CPP_INLINE_VARIABLES
 constexpr OUStringLiteral AnimExpImpl::gsDimColor;
 constexpr OUStringLiteral AnimExpImpl::gsDimHide;
 constexpr OUStringLiteral AnimExpImpl::gsDimPrev;
@@ -254,7 +253,6 @@ constexpr OUStringLiteral AnimExpImpl::g
 constexpr OUStringLiteral AnimExpImpl::gsTextEffect;
 constexpr OUStringLiteral AnimExpImpl::gsIsAnimation;
 constexpr OUStringLiteral AnimExpImpl::gsAnimPath;
-#endif
 
 XMLAnimationsExporter::XMLAnimationsExporter( XMLShapeExport* pShapeExp )
     : mpImpl( new AnimExpImpl )
diff -up libreoffice-6.2.0/xmloff/source/draw/animimp.cxx.omv~ libreoffice-6.2.0/xmloff/source/draw/animimp.cxx
--- libreoffice-6.2.0/xmloff/source/draw/animimp.cxx.omv~	2018-12-14 20:16:22.393681280 +0100
+++ libreoffice-6.2.0/xmloff/source/draw/animimp.cxx	2018-12-14 20:16:26.166732024 +0100
@@ -335,7 +335,6 @@ public:
     static constexpr OUStringLiteral gsIsAnimation = "IsAnimation";
 };
 
-#if !HAVE_CPP_INLINE_VARIABLES
 constexpr OUStringLiteral AnimImpImpl::gsDimColor;
 constexpr OUStringLiteral AnimImpImpl::gsDimHide;
 constexpr OUStringLiteral AnimImpImpl::gsDimPrev;
@@ -348,7 +347,6 @@ constexpr OUStringLiteral AnimImpImpl::g
 constexpr OUStringLiteral AnimImpImpl::gsPresShapeService;
 constexpr OUStringLiteral AnimImpImpl::gsAnimPath;
 constexpr OUStringLiteral AnimImpImpl::gsIsAnimation;
-#endif
 
 enum XMLActionKind
 {
diff -up libreoffice-6.2.0/xmloff/source/draw/sdxmlimp.cxx.omv~ libreoffice-6.2.0/xmloff/source/draw/sdxmlimp.cxx
--- libreoffice-6.2.0/xmloff/source/draw/sdxmlimp.cxx.omv~	2018-12-14 20:16:40.922930654 +0100
+++ libreoffice-6.2.0/xmloff/source/draw/sdxmlimp.cxx	2018-12-14 20:16:43.627967092 +0100
@@ -288,10 +288,8 @@ SERVICE( XMLDrawMetaImportOasis, "com.su
 SERVICE( XMLImpressSettingsImportOasis, "com.sun.star.comp.Impress.XMLOasisSettingsImporter", "XMLImpressSettingsImportOasis", false, SvXMLImportFlags::SETTINGS )
 SERVICE( XMLDrawSettingsImportOasis, "com.sun.star.comp.Draw.XMLOasisSettingsImporter", "XMLImpressSettingsImportOasis", true, SvXMLImportFlags::SETTINGS )
 
-#if !HAVE_CPP_INLINE_VARIABLES
 constexpr OUStringLiteral SdXMLImport::gsPageLayouts;
 constexpr OUStringLiteral SdXMLImport::gsPreview;
-#endif
 
 SdXMLImport::SdXMLImport(
     const css::uno::Reference< css::uno::XComponentContext >& xContext,
diff -up libreoffice-6.2.0/xmloff/source/text/XMLIndexMarkExport.cxx.omv~ libreoffice-6.2.0/xmloff/source/text/XMLIndexMarkExport.cxx
--- libreoffice-6.2.0/xmloff/source/text/XMLIndexMarkExport.cxx.omv~	2018-12-14 20:16:55.503127150 +0100
+++ libreoffice-6.2.0/xmloff/source/text/XMLIndexMarkExport.cxx	2018-12-14 20:16:59.778184807 +0100
@@ -38,7 +38,6 @@ using ::com::sun::star::uno::Reference;
 using ::com::sun::star::uno::Any;
 
 
-#if !HAVE_CPP_INLINE_VARIABLES
 constexpr OUStringLiteral XMLIndexMarkExport::gsLevel;
 constexpr OUStringLiteral XMLIndexMarkExport::gsUserIndexName;
 constexpr OUStringLiteral XMLIndexMarkExport::gsPrimaryKey;
@@ -51,7 +50,6 @@ constexpr OUStringLiteral XMLIndexMarkEx
 constexpr OUStringLiteral XMLIndexMarkExport::gsPrimaryKeyReading;
 constexpr OUStringLiteral XMLIndexMarkExport::gsSecondaryKeyReading;
 constexpr OUStringLiteral XMLIndexMarkExport::gsMainEntry;
-#endif
 
 
 XMLIndexMarkExport::XMLIndexMarkExport(
diff -up libreoffice-6.2.0/sc/source/ui/Accessibility/AccessibleDocument.cxx.omv~ libreoffice-6.2.0/sc/source/ui/Accessibility/AccessibleDocument.cxx
--- libreoffice-6.2.0/sc/source/ui/Accessibility/AccessibleDocument.cxx.omv~	2018-12-14 23:24:22.226673683 +0100
+++ libreoffice-6.2.0/sc/source/ui/Accessibility/AccessibleDocument.cxx	2018-12-14 23:24:30.859800967 +0100
@@ -192,10 +192,8 @@ struct ScShapeDataLess
     }
 };
 
-#if !HAVE_CPP_INLINE_VARIABLES
 constexpr OUStringLiteral ScShapeDataLess::gsLayerId;
 constexpr OUStringLiteral ScShapeDataLess::gsZOrder;
-#endif
 
 struct DeselectShape
 {
