--- libreoffice-4.4.2/vcl/unx/gtk/a11y/atktextattributes.cxx.omv~	2015-04-03 00:02:30.198147803 +0200
+++ libreoffice-4.4.2/vcl/unx/gtk/a11y/atktextattributes.cxx	2015-04-03 00:02:35.254167939 +0200
@@ -1235,7 +1235,7 @@ attribute_set_new_from_extended_attribut
 
         pSet = attribute_set_prepend( pSet,
                                       atk_text_attribute_register( sPropertyName.getStr() ),
-                                      g_strdup_printf( sPropertyValue.getStr() ) );
+                                      g_strdup_printf( "%s", sPropertyValue.getStr() ) );
     }
     while ( nIndex >= 0 && nIndex < sExtendedAttrs.getLength() );
 
