diff -up libreoffice-24.2.0.0/configure.ac.omv~ libreoffice-24.2.0.0/configure.ac
--- libreoffice-24.2.0.0/configure.ac.omv~	2023-11-30 22:10:30.577248462 +0100
+++ libreoffice-24.2.0.0/configure.ac	2023-11-30 22:10:58.133520073 +0100
@@ -5726,7 +5726,6 @@ if test "$cross_compiling" = "yes"; then
         sysui/desktop/macosx/Info.plist.in \
         sysui/desktop/macosx/hardened_runtime.xcent.in \
         sysui/desktop/macosx/lo.xcent.in \
-        .vscode/vs-code-template.code-workspace.in \
         solenv/lockfile/autoconf.h.in \
         ) \
     | (cd CONF-FOR-BUILD && tar xf -)
@@ -15085,11 +15084,7 @@ AC_CONFIG_FILES([config_host.mk
                  bin/bffvalidator.sh
                  bin/odfvalidator.sh
                  bin/officeotron.sh
-                 instsetoo_native/util/openoffice.lst
-                 sysui/desktop/macosx/Info.plist
-                 hardened_runtime.xcent:sysui/desktop/macosx/hardened_runtime.xcent.in
-                 lo.xcent:sysui/desktop/macosx/lo.xcent.in
-                 vs-code.code-workspace.template:.vscode/vs-code-template.code-workspace.in])
+                 instsetoo_native/util/openoffice.lst])
 
 AC_CONFIG_HEADERS([config_host/config_atspi.h])
 AC_CONFIG_HEADERS([config_host/config_buildconfig.h])
