diff -up libreoffice-6.3.0/configure.ac.3~ libreoffice-6.3.0/configure.ac
--- libreoffice-6.3.0/configure.ac.3~	2019-08-12 11:05:07.970767186 +0200
+++ libreoffice-6.3.0/configure.ac	2019-08-12 11:07:59.468050253 +0200
@@ -7256,8 +7256,8 @@ dnl Checks for javac
 dnl ===================================================================
 if test "$ENABLE_JAVA" != ""; then
     javacompiler="javac"
-    : ${JAVA_SOURCE_VER=1.6}
-    : ${JAVA_TARGET_VER=1.6}
+    : ${JAVA_SOURCE_VER=7}
+    : ${JAVA_TARGET_VER=7}
     if test -z "$with_jdk_home"; then
         AC_PATH_PROG(JAVACOMPILER, $javacompiler)
     else
diff -up libreoffice-6.3.0/external/languagetool/languagetool-java12.patch.3~ libreoffice-6.3.0/external/languagetool/languagetool-java12.patch
--- libreoffice-6.3.0/external/languagetool/languagetool-java12.patch.3~	2019-08-12 11:05:07.971767188 +0200
+++ libreoffice-6.3.0/external/languagetool/languagetool-java12.patch	2019-08-12 11:05:07.971767188 +0200
@@ -0,0 +1,11 @@
+diff -up workdir/UnpackedTarball/languagetool/build.properties.omv~ workdir/UnpackedTarball/languagetool/build.properties
+--- workdir/UnpackedTarball/languagetool/build.properties.omv~	2019-04-19 02:32:20.478922142 +0200
++++ workdir/UnpackedTarball/languagetool/build.properties	2019-04-19 02:32:24.828921931 +0200
+@@ -77,5 +77,5 @@ build.meta-inf = ${build.dir}/META-INF
+ final.name = LanguageTool
+ final.name.gui = LanguageToolGUI
+ encoding = utf-8
+-target = 1.6
+-source = 1.6
++target = 1.7
++source = 1.7
diff -up libreoffice-6.3.0/external/languagetool/UnpackedTarball_languagetool.mk.3~ libreoffice-6.3.0/external/languagetool/UnpackedTarball_languagetool.mk
--- libreoffice-6.3.0/external/languagetool/UnpackedTarball_languagetool.mk.3~	2019-08-06 19:26:35.000000000 +0200
+++ libreoffice-6.3.0/external/languagetool/UnpackedTarball_languagetool.mk	2019-08-12 11:05:07.971767188 +0200
@@ -20,6 +20,7 @@ $(eval $(call gb_UnpackedTarball_add_pat
     external/languagetool/english.ireland.patch \
     external/languagetool/esperanto.territory.patch \
     external/languagetool/french.haiti.patch \
+    external/languagetool/languagetool-java12.patch \
 ))
 
 # vim: set noet sw=4 ts=4:
