diff -up libreoffice-6.2.0/sfx2/source/appl/shutdownicon.cxx.omv~ libreoffice-6.2.0/sfx2/source/appl/shutdownicon.cxx
--- libreoffice-6.2.0/sfx2/source/appl/shutdownicon.cxx.omv~	2018-12-17 06:45:51.059059002 +0100
+++ libreoffice-6.2.0/sfx2/source/appl/shutdownicon.cxx	2018-12-17 06:50:02.681676738 +0100
@@ -142,7 +142,7 @@ bool LoadModule()
 #endif // ENABLE_QUICKSTART_APPLET
     }
     assert(!boost::logic::indeterminate(loaded));
-    return loaded;
+    return static_cast<bool>(loaded);
 }
 
 }
