diff -up libreoffice-7.1.0/i18npool/source/calendar/calendar_gregorian.cxx.omv~ libreoffice-7.1.0/i18npool/source/calendar/calendar_gregorian.cxx
--- libreoffice-7.1.0/i18npool/source/calendar/calendar_gregorian.cxx.omv~	2020-10-28 14:22:57.424826430 +0100
+++ libreoffice-7.1.0/i18npool/source/calendar/calendar_gregorian.cxx	2020-10-28 14:23:02.885897583 +0100
@@ -360,7 +360,7 @@ Calendar_gregorian::setLocalDateTime( do
             "Calendar_gregorian::setLocalDateTime: " << std::fixed << fM << " rounded to " << fR);
     int32_t nZoneOffset, nDSTOffset;
     UErrorCode status = U_ZERO_ERROR;
-    body->getTimeZone().getOffset( fR, TRUE, nZoneOffset, nDSTOffset, status );
+    body->getTimeZone().getOffset( fR, true, nZoneOffset, nDSTOffset, status );
     if ( !U_SUCCESS(status) ) throw ERROR;
     status = U_ZERO_ERROR;
     body->setTime( fR - (nZoneOffset + nDSTOffset), status );
