diff -up libreoffice-7.4.0.1/comphelper/source/misc/threadpool.cxx.8~ libreoffice-7.4.0.1/comphelper/source/misc/threadpool.cxx
--- libreoffice-7.4.0.1/comphelper/source/misc/threadpool.cxx.8~	2022-07-06 21:17:19.000000000 +0200
+++ libreoffice-7.4.0.1/comphelper/source/misc/threadpool.cxx	2022-07-07 04:12:22.402566116 +0200
@@ -55,18 +55,19 @@ public:
 };
 
 
-class ThreadPool::ThreadWorker : public salhelper::Thread
+class ThreadPool::ThreadWorker : public salhelper::SimpleReferenceObject //salhelper::Thread
 {
     ThreadPool *mpPool;
+    std::thread mThread;
 public:
 
     explicit ThreadWorker( ThreadPool *pPool ) :
-        salhelper::Thread("thread-pool"),
+        //salhelper::Thread("thread-pool"),
         mpPool( pPool )
     {
     }
 
-    virtual void execute() override
+    virtual void execute() //override
     {
 #if defined DBG_UTIL && (defined LINUX || defined _WIN32)
         gbIsWorkerThread = true;
@@ -91,6 +92,16 @@ public:
             }
         }
     }
+
+    virtual void launch()
+    {
+        mThread = std::thread(&ThreadWorker::execute, this);
+    }
+
+    virtual void join()
+    {
+        mThread.join();
+    }
 };
 
 ThreadPool::ThreadPool(std::size_t nWorkers)
