diff -up libreoffice-7.4.0.0/vcl/qt5/QtFrame.cxx.6~ libreoffice-7.4.0.0/vcl/qt5/QtFrame.cxx
--- libreoffice-7.4.0.0/vcl/qt5/QtFrame.cxx.6~	2022-05-12 15:26:05.000000000 +0200
+++ libreoffice-7.4.0.0/vcl/qt5/QtFrame.cxx	2022-06-03 17:17:29.849119089 +0200
@@ -684,7 +684,11 @@ void QtFrame::SetWindowState(const SalFr
     {
         const qreal fRatio = devicePixelRatioF();
         QWidget* const pChild = asChild();
-        pChild->resize(ceil(pState->mnWidth / fRatio), ceil(pState->mnHeight / fRatio));
+        // Workaround for LO appearing way too small in some situations, e.g.
+        // https://github.com/QubesOS/qubes-issues/issues/3281
+        int w = ceil(pState->mnWidth / fRatio), h = ceil(pState->mnHeight / fRatio);
+        if(w > 10 && h > 10 && w < 50000 && h < 50000)
+            pChild->resize(w, h);
         pChild->move(ceil(pState->mnX / fRatio), ceil(pState->mnY / fRatio));
         SetWindowStateImpl(Qt::WindowMaximized);
     }
@@ -701,7 +705,10 @@ void QtFrame::SetWindowState(const SalFr
             nPosSizeFlags |= SAL_FRAME_POSSIZE_WIDTH;
         if (pState->mnMask & WindowStateMask::Height)
             nPosSizeFlags |= SAL_FRAME_POSSIZE_HEIGHT;
-        SetPosSize(pState->mnX, pState->mnY, pState->mnWidth, pState->mnHeight, nPosSizeFlags);
+        // Workaround for LO appearing way too small in some situations, e.g.
+        // https://github.com/QubesOS/qubes-issues/issues/3281
+        if(pState->mnWidth > 10 && pState->mnWidth < 50000 && pState->mnHeight > 10 && pState->mnHeight < 50000)
+            SetPosSize(pState->mnX, pState->mnY, pState->mnWidth, pState->mnHeight, nPosSizeFlags);
     }
     else if (pState->mnMask & WindowStateMask::State && !isChild())
     {
diff -up libreoffice-7.4.0.0/vcl/unx/generic/window/salframe.cxx.6~ libreoffice-7.4.0.0/vcl/unx/generic/window/salframe.cxx
--- libreoffice-7.4.0.0/vcl/unx/generic/window/salframe.cxx.6~	2022-05-12 15:26:05.000000000 +0200
+++ libreoffice-7.4.0.0/vcl/unx/generic/window/salframe.cxx	2022-06-03 17:20:59.898240260 +0200
@@ -1659,9 +1659,12 @@ void X11SalFrame::SetWindowState( const
                                pHints );
             XFree( pHints );
 
-            XMoveResizeWindow( GetXDisplay(), GetShellWindow(),
-                               pState->mnX, pState->mnY,
-                               pState->mnWidth, pState->mnHeight );
+            // Workaround for LO appearing way too small in some situations, e.g.
+            // https://github.com/QubesOS/qubes-issues/issues/3281
+            if(pState->mnWidth > 10 && pState->mnWidth < 50000 && pState->mnHeight > 10 && pState->mnHeight < 50000)
+                XMoveResizeWindow( GetXDisplay(), GetShellWindow(),
+                                   pState->mnX, pState->mnY,
+                                   pState->mnWidth, pState->mnHeight );
             // guess maximized geometry from last time
             maGeometry.nX      = pState->mnMaximizedX;
             maGeometry.nY      = pState->mnMaximizedY;
@@ -1742,10 +1745,13 @@ void X11SalFrame::SetWindowState( const
                     aPosSize.Move( 0, static_cast<tools::Long>(aGeom.nTopDecoration) - nTop );
             }
 
-            SetPosSize(aPosSize.getX(), aPosSize.getY(),
-                       aPosSize.GetWidth(), aPosSize.GetHeight(),
-                       SAL_FRAME_POSSIZE_WIDTH | SAL_FRAME_POSSIZE_HEIGHT |
-                       nPosFlags);
+            // Workaround for LO appearing way too small in some situations, e.g.
+            // https://github.com/QubesOS/qubes-issues/issues/3281
+            if(aPosSize.GetWidth() > 10 && aPosSize.GetWidth() < 50000 && aPosSize.GetHeight() > 10 && aPosSize.GetHeight() < 50000)
+                SetPosSize(aPosSize.getX(), aPosSize.getY(),
+                           aPosSize.GetWidth(), aPosSize.GetHeight(),
+                           SAL_FRAME_POSSIZE_WIDTH | SAL_FRAME_POSSIZE_HEIGHT |
+                           nPosFlags);
         }
     }
 
