--- libreoffice-4.1.0/configure.ac.omv~	2013-08-30 00:27:27.892982601 +0200
+++ libreoffice-4.1.0/configure.ac	2013-08-30 00:27:47.593610076 +0200
@@ -7216,7 +7216,7 @@ libo_CHECK_SYSTEM_MODULE([libvisio],[VIS
 dnl ===================================================================
 dnl Check for system libcmis
 dnl ===================================================================
-libo_CHECK_SYSTEM_MODULE([libcmis],[CMIS],[libcmis-0.3 >= 0.3.1])
+libo_CHECK_SYSTEM_MODULE([libcmis],[CMIS],[libcmis-0.4])
 
 dnl ===================================================================
 dnl Check for system libwpd
--- libreoffice-4.1.0/configure.omv~	2013-08-30 00:27:50.498555145 +0200
+++ libreoffice-4.1.0/configure	2013-08-30 00:28:51.163407981 +0200
@@ -18803,12 +18803,12 @@ if test -n "$CMIS_CFLAGS"; then
     pkg_cv_CMIS_CFLAGS="$CMIS_CFLAGS"
  elif test -n "$PKG_CONFIG"; then
     if test -n "$PKG_CONFIG" && \
-    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"libcmis-0.3 >= 0.3.1\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "libcmis-0.3 >= 0.3.1") 2>&5
+    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"libcmis-0.4\""; } >&5
+  ($PKG_CONFIG --exists --print-errors "libcmis-0.4") 2>&5
   ac_status=$?
   $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; then
-  pkg_cv_CMIS_CFLAGS=`$PKG_CONFIG --cflags "libcmis-0.3 >= 0.3.1" 2>/dev/null`
+  pkg_cv_CMIS_CFLAGS=`$PKG_CONFIG --cflags "libcmis-0.4" 2>/dev/null`
 		      test "x$?" != "x0" && pkg_failed=yes
 else
   pkg_failed=yes
@@ -18820,12 +18820,12 @@ if test -n "$CMIS_LIBS"; then
     pkg_cv_CMIS_LIBS="$CMIS_LIBS"
  elif test -n "$PKG_CONFIG"; then
     if test -n "$PKG_CONFIG" && \
-    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"libcmis-0.3 >= 0.3.1\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "libcmis-0.3 >= 0.3.1") 2>&5
+    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"libcmis-0.4\""; } >&5
+  ($PKG_CONFIG --exists --print-errors "libcmis-0.4") 2>&5
   ac_status=$?
   $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; then
-  pkg_cv_CMIS_LIBS=`$PKG_CONFIG --libs "libcmis-0.3 >= 0.3.1" 2>/dev/null`
+  pkg_cv_CMIS_LIBS=`$PKG_CONFIG --libs "libcmis-0.4" 2>/dev/null`
 		      test "x$?" != "x0" && pkg_failed=yes
 else
   pkg_failed=yes
@@ -18846,14 +18846,14 @@ else
         _pkg_short_errors_supported=no
 fi
         if test $_pkg_short_errors_supported = yes; then
-	        CMIS_PKG_ERRORS=`$PKG_CONFIG --short-errors --print-errors --cflags --libs "libcmis-0.3 >= 0.3.1" 2>&1`
+	        CMIS_PKG_ERRORS=`$PKG_CONFIG --short-errors --print-errors --cflags --libs "libcmis-0.4" 2>&1`
         else
-	        CMIS_PKG_ERRORS=`$PKG_CONFIG --print-errors --cflags --libs "libcmis-0.3 >= 0.3.1" 2>&1`
+	        CMIS_PKG_ERRORS=`$PKG_CONFIG --print-errors --cflags --libs "libcmis-0.4" 2>&1`
         fi
 	# Put the nasty error message in config.log where it belongs
 	echo "$CMIS_PKG_ERRORS" >&5
 
-	as_fn_error $? "Package requirements (libcmis-0.3 >= 0.3.1) were not met:
+	as_fn_error $? "Package requirements (libcmis-0.4) were not met:
 
 $CMIS_PKG_ERRORS
 
--- libreoffice-4.1.0/ucb/source/ucp/cmis/auth_provider.hxx.omv~	2013-08-30 03:41:06.437325736 +0200
+++ libreoffice-4.1.0/ucb/source/ucp/cmis/auth_provider.hxx	2013-08-30 03:42:57.375242399 +0200
@@ -13,6 +13,7 @@
 #define AUTH_PROVIDER_HXX
 
 #include <libcmis/session.hxx>
+#include <libcmis/session-factory.hxx>
 
 #include <com/sun/star/ucb/XCommandEnvironment.hpp>
 
--- libreoffice-4.1.0/ucb/source/ucp/cmis/cmis_repo_content.cxx.omv~	2013-08-30 03:50:25.248828468 +0200
+++ libreoffice-4.1.0/ucb/source/ucp/cmis/cmis_repo_content.cxx	2013-08-30 04:04:47.238571797 +0200
@@ -43,7 +43,7 @@ namespace cmis
 {
     RepoContent::RepoContent( const uno::Reference< uno::XComponentContext >& rxContext,
         ContentProvider *pProvider, const uno::Reference< ucb::XContentIdentifier >& Identifier,
-        list< libcmis::RepositoryPtr > aRepos )
+        vector< libcmis::RepositoryPtr > aRepos )
             throw ( ucb::ContentCreationException )
         : ContentImplHelper( rxContext, pProvider, Identifier ),
         m_pProvider( pProvider ),
--- libreoffice-4.1.0/ucb/source/ucp/cmis/cmis_repo_content.hxx.omv~	2013-08-30 03:50:48.639388911 +0200
+++ libreoffice-4.1.0/ucb/source/ucp/cmis/cmis_repo_content.hxx	2013-08-30 03:51:45.660317315 +0200
@@ -26,6 +26,7 @@
 #include <libcmis/repository.hxx>
 
 #include <list>
+#include <vector>
 
 namespace com { namespace sun { namespace star {
     namespace beans {
@@ -55,7 +56,7 @@ private:
     URL                    m_aURL;
     OUString          m_sRepositoryId;
 
-    std::list< libcmis::RepositoryPtr > m_aRepositories;
+    std::vector< libcmis::RepositoryPtr > m_aRepositories;
 
 private:
 
@@ -80,7 +80,7 @@ public:
     RepoContent( const com::sun::star::uno::Reference<
         com::sun::star::uno::XComponentContext >& rxContext, ContentProvider *pProvider,
         const com::sun::star::uno::Reference< com::sun::star::ucb::XContentIdentifier >& Identifier,
-        std::list< libcmis::RepositoryPtr > aRepos = std::list< libcmis::RepositoryPtr > ( ) )
+        std::vector< libcmis::RepositoryPtr > aRepos = std::vector< libcmis::RepositoryPtr > ( ) )
             throw ( com::sun::star::ucb::ContentCreationException );
 
     virtual ~RepoContent();
@@ -172,7 +172,7 @@ namespace cmis
 
         if ( !m_sRepositoryId.isEmpty() )
         {
-            for ( list< libcmis::RepositoryPtr >::iterator it = m_aRepositories.begin( );
+            for ( vector< libcmis::RepositoryPtr >::iterator it = m_aRepositories.begin( );
                     it != m_aRepositories.end( ) && NULL == repo.get( ); ++it )
             {
                 if ( STD_TO_OUSTR( ( *it )->getId( ) ) == m_sRepositoryId )
@@ -352,7 +352,7 @@ namespace cmis
 
         if ( m_sRepositoryId.isEmpty( ) )
         {
-            for ( list< libcmis::RepositoryPtr >::iterator it = m_aRepositories.begin( );
+            for ( vector< libcmis::RepositoryPtr >::iterator it = m_aRepositories.begin( );
                     it != m_aRepositories.end(); ++it )
             {
                 URL aUrl( m_aURL );
