diff -up libreoffice-7.4.0.1/vcl/qt5/QtFrame.cxx.6~ libreoffice-7.4.0.1/vcl/qt5/QtFrame.cxx
--- libreoffice-7.4.0.1/vcl/qt5/QtFrame.cxx.6~	2022-07-07 04:02:51.544975546 +0200
+++ libreoffice-7.4.0.1/vcl/qt5/QtFrame.cxx	2022-07-07 04:06:36.950870891 +0200
@@ -699,7 +699,13 @@ void QtFrame::SetWindowState(const vcl::
     {
         const qreal fRatio = devicePixelRatioF();
         QWidget* const pChild = asChild();
-        pChild->resize(ceil(pState->width() / fRatio), ceil(pState->height() / fRatio));
+
+        // Workaround for LO appearing way too small in some situations, e.g.
+        // https://github.com/QubesOS/qubes-issues/issues/3281
+        int w = ceil(pState->width() / fRatio), h = ceil(pState->height() / fRatio);
+        if(w > 10 && h > 10 && w < 50000 && h < 50000)
+            pChild->resize(w, h);
+
         pChild->move(ceil(pState->x() / fRatio), ceil(pState->y() / fRatio));
         SetWindowStateImpl(Qt::WindowMaximized);
     }
@@ -714,7 +720,10 @@ void QtFrame::SetWindowState(const vcl::
             nPosSizeFlags |= SAL_FRAME_POSSIZE_WIDTH;
         if (pState->mask() & vcl::WindowDataMask::Height)
             nPosSizeFlags |= SAL_FRAME_POSSIZE_HEIGHT;
-        SetPosSize(pState->x(), pState->y(), pState->width(), pState->height(), nPosSizeFlags);
+        // Workaround for LO appearing way too small in some situations, e.g.
+        // https://github.com/QubesOS/qubes-issues/issues/3281
+        if(pState->width() > 10 && pState->width() < 50000 && pState->height() > 10 && pState->height() < 50000)
+            SetPosSize(pState->x(), pState->y(), pState->width(), pState->height(), nPosSizeFlags);
     }
     else if (pState->mask() & vcl::WindowDataMask::State && !isChild())
     {
diff -up libreoffice-7.4.0.1/vcl/unx/generic/window/salframe.cxx.6~ libreoffice-7.4.0.1/vcl/unx/generic/window/salframe.cxx
--- libreoffice-7.4.0.1/vcl/unx/generic/window/salframe.cxx.6~	2022-07-07 04:02:51.545975524 +0200
+++ libreoffice-7.4.0.1/vcl/unx/generic/window/salframe.cxx	2022-07-07 04:09:28.740982174 +0200
@@ -1635,8 +1635,12 @@ void X11SalFrame::SetWindowState( const
             XSetWMNormalHints(GetXDisplay(), GetShellWindow(), pHints);
             XFree( pHints );
 
-            XMoveResizeWindow(GetXDisplay(), GetShellWindow(), pState->x(), pState->y(),
-                              pState->width(), pState->height());
+            // Workaround for LO appearing way too small in some situations, e.g.
+            // https://github.com/QubesOS/qubes-issues/issues/3281
+            if(pState->width() > 10 && pState->width() < 50000 && pState->height() > 10 && pState->height() < 50000)
+                XMoveResizeWindow( GetXDisplay(), GetShellWindow(),
+                                   pState->x(), pState->y(),
+                                   pState->width(), pState->height() );
             // guess maximized geometry from last time
             maGeometry.setPos({ pState->GetMaximizedX(), pState->GetMaximizedY() });
             maGeometry.setSize({ pState->GetMaximizedWidth(), pState->GetMaximizedHeight() });
@@ -1707,10 +1711,13 @@ void X11SalFrame::SetWindowState( const
                     aPosSize.Move( 0, static_cast<tools::Long>(aGeom.topDecoration()) - nTop );
             }
 
-            SetPosSize(aPosSize.getX(), aPosSize.getY(),
-                       aPosSize.GetWidth(), aPosSize.GetHeight(),
-                       SAL_FRAME_POSSIZE_WIDTH | SAL_FRAME_POSSIZE_HEIGHT |
-                       nPosFlags);
+            // Workaround for LO appearing way too small in some situations, e.g.
+            // https://github.com/QubesOS/qubes-issues/issues/3281
+            if(aPosSize.GetWidth() > 10 && aPosSize.GetWidth() < 50000 && aPosSize.GetHeight() > 10 && aPosSize.GetHeight() < 50000)
+                SetPosSize(aPosSize.getX(), aPosSize.getY(),
+                           aPosSize.GetWidth(), aPosSize.GetHeight(),
+                           SAL_FRAME_POSSIZE_WIDTH | SAL_FRAME_POSSIZE_HEIGHT |
+                           nPosFlags);
         }
     }
 
