diff -up libreoffice-7.1.1/avmedia/source/vlc/vlcplayer.cxx.omv~ libreoffice-7.1.1/avmedia/source/vlc/vlcplayer.cxx
diff -up libreoffice-7.1.1/include/vcl/sysdata.hxx.omv~ libreoffice-7.1.1/include/vcl/sysdata.hxx
--- libreoffice-7.1.1/include/vcl/sysdata.hxx.omv~	2021-03-06 04:36:36.318993426 +0100
+++ libreoffice-7.1.1/include/vcl/sysdata.hxx	2021-03-06 04:36:58.702388150 +0100
@@ -73,7 +73,7 @@ struct VCL_DLLPUBLIC SystemEnvData
     // however, the GTK3 vclplug wants to store pointers in here!
     sal_IntPtr          aShellWindow;   // the window of the frame's shell
     Platform            platform;       // the windowing system in use
-private:
+//private: // can't be private because it's used in vlc
     sal_uIntPtr         aWindow;        // the window of the object
 public:
 
