diff -up libreoffice-7.4.0.0/configure.ac.3~ libreoffice-7.4.0.0/configure.ac
--- libreoffice-7.4.0.0/configure.ac.3~	2022-06-03 17:13:24.258766264 +0200
+++ libreoffice-7.4.0.0/configure.ac	2022-06-03 17:16:17.054723709 +0200
@@ -5606,7 +5606,6 @@ if test "$cross_compiling" = "yes"; then
         instsetoo_native/util/openoffice.lst.in \
         config_host/*.in \
         sysui/desktop/macosx/Info.plist.in \
-        .vscode/vs-code-template.code-workspace.in \
         solenv/lockfile/autoconf.h.in \
         ) \
     | (cd CONF-FOR-BUILD && tar xf -)
@@ -14656,8 +14655,7 @@ AC_CONFIG_FILES([config_host.mk
                  bin/officeotron.sh
                  hardened_runtime.xcent
                  instsetoo_native/util/openoffice.lst
-                 sysui/desktop/macosx/Info.plist
-                 vs-code.code-workspace.template:.vscode/vs-code-template.code-workspace.in])
+                 sysui/desktop/macosx/Info.plist])
 AC_CONFIG_HEADERS([config_host/config_buildid.h])
 AC_CONFIG_HEADERS([config_host/config_box2d.h])
 AC_CONFIG_HEADERS([config_host/config_clang.h])
