diff -uraN libreoffice-26.8.0.1/oox/source/export/shapes.cxx omv-libreoffice-26.8.0.1/oox/source/export/shapes.cxx
--- libreoffice-26.8.0.1/oox/source/export/shapes.cxx	2026-07-13 11:47:46.000000000 +0100
+++ omv-libreoffice-26.8.0.1/oox/source/export/shapes.cxx	2026-07-20 15:26:02.935125185 +0100
@@ -2252,7 +2252,7 @@
         return false;
 
     const OUString sShapeType = xShape->getShapeType();
-    return constMap.contains(sShapeType);
+    return constMap.find(sShapeType) != constMap.end();
 }
 
 bool ShapeExport::IsValidShape(const Reference<XShape>& xShape, DocumentType eDocumentType)

