--- libreoffice-5.4.0/vcl/unx/kde4/KDESalGraphics.cxx.0051~	2017-06-24 23:59:34.599570618 +0200
+++ libreoffice-5.4.0/vcl/unx/kde4/KDESalGraphics.cxx	2017-06-25 00:01:32.470770037 +0200
@@ -138,8 +138,13 @@ namespace
         option->state |= state;
         option->rect = image->rect();
 
-        QPainter painter(image);
-        QApplication::style()->drawComplexControl(element, option, &painter);
+        // Hack: Don't display tiny useless scrollbars
+        // Impress is known to crash when adding effects (segfault is triggered by 15x18 scrollbar)
+        if (!((option->type == QStyleOption::SO_Slider) && ((option->rect.width() * option->rect.height()) < 300)))
+        {
+            QPainter painter(image);
+            QApplication::style()->drawComplexControl(element, option, &painter);
+        }
     }
 
     void lcl_drawFrame( QStyle::PrimitiveElement element, QImage* image, QStyle::State const & state,
