--- libreoffice-4.1.2/bin/distro-install-file-lists.0100~	2013-09-29 01:20:00.000000000 +0200
+++ libreoffice-4.1.2/bin/distro-install-file-lists	2013-10-30 17:42:31.090856924 +0100
@@ -135,14 +135,18 @@ if test "z$OOO_VENDOR" != "zDebian" ; th
     fi
 
     if test "$SPLIT_OPT_FEATURES" = "YES" ; then
-        if test "z$OOO_VENDOR" = "zMandriva" ; then
+        if test "z$OOO_VENDOR" = "zMandriva" -o "z$OOO_VENDOR" = "zOpenMandriva" -o "z$OOO_VENDOR" = "zROSA" ; then
             merge_flists gid_Module_Optional_Grfflt            $FILELISTSDIR/draw_list.txt
             merge_flists gid_Module_Optional_Headless          $FILELISTSDIR/common_list.txt
-            merge_flists gid_Module_Optional_Pymailmerge       $FILELISTSDIR/pyuno_list.txt
             merge_flists gid_Module_Pyuno                      $FILELISTSDIR/pyuno_list.txt
             merge_flists gid_Module_Script_Provider_For_Python $FILELISTSDIR/pyuno_list.txt
             merge_flists gid_Module_Optional_Pyuno_LibreLogo   $FILELISTSDIR/pyuno_list.txt
-            merge_flists gid_Module_Optional_Xsltfiltersamples $FILELISTSDIR/common_list.txt
+            if test "z$OOO_VENDOR" = "zOpenMandriva"; then
+                merge_flists gid_Module_Optional_Pymailmerge       $FILELISTSDIR/mailmerge_list.txt
+            else
+                merge_flists gid_Module_Optional_Xsltfiltersamples $FILELISTSDIR/common_list.txt
+                merge_flists gid_Module_Optional_Pymailmerge       $FILELISTSDIR/pyuno_list.txt
+            fi
         else
             merge_flists gid_Module_Optional_Grfflt            $FILELISTSDIR/common_list.txt
             merge_flists gid_Module_Optional_Headless          $FILELISTSDIR/common_list.txt
@@ -165,7 +169,7 @@ if test "z$OOO_VENDOR" != "zDebian" ; th
     # lang packs
     for lang in `echo $WITH_LANG_LIST | sed -e s/-/_/g`; do
         lang_lists=
-        if test "$OOO_VENDOR" = "Mandriva" -o \( "$OOO_VENDOR" = "SUSE" -a "$SPLIT_APP_MODULES" = "YES" \) ; then
+        if test "$OOO_VENDOR" = "Mandriva" -o "$OOO_VENDOR" = "OpenMandriva" -o "$OOO_VENDOR" = "ROSA" -o \( "$OOO_VENDOR" = "SUSE" -a "$SPLIT_APP_MODULES" = "YES" \) ; then
             test -f gid_Module_Langpack_Basis_$lang     && lang_lists="$lang_lists gid_Module_Langpack_Basis_$lang" || :
             test -f gid_Module_Langpack_Brand_$lang     && lang_lists="$lang_lists gid_Module_Langpack_Brand_$lang" || :
             test -f gid_Module_Langpack_Resource_$lang  && lang_lists="$lang_lists gid_Module_Langpack_Resource_$lang" || :
@@ -196,7 +200,7 @@ if test "z$OOO_VENDOR" != "zDebian" ; th
         fi
         # some help files are in _Langpack_{Writer,Impress,...}_<lang>
         # move them from -l10n to -help
-        if test "$OOO_VENDOR" = "Mandriva" -o \( "$OOO_VENDOR" = "SUSE" -a "$SPLIT_APP_MODULES" = "YES" \) ; then
+        if test "$OOO_VENDOR" = "Mandriva" -o "$OOO_VENDOR" = "OpenMandriva" -o "$OOO_VENDOR" = "ROSA" -o \( "$OOO_VENDOR" = "SUSE" -a "$SPLIT_APP_MODULES" = "YES" \) ; then
             for lang in `echo $WITH_LANG_LIST | sed -e s/-/_/g`; do
                 test -f $FILELISTSDIR/help_${lang}_list.txt || continue;
                 mv_file_between_flists $FILELISTSDIR/help_${lang}_list.txt $FILELISTSDIR/lang_${lang}_list.txt $INSTALLDIR/help/.*
@@ -264,7 +268,7 @@ if test "z$OOO_VENDOR" != "zDebian" ; th
 
     # mailmerge
     if test "$SPLIT_OPT_FEATURES" = "YES" ; then
-        if test "z$OOO_VENDOR" = "zMandriva" ; then
+        if test "z$OOO_VENDOR" = "zMandriva" -o "z$OOO_VENDOR" = "zROSA" ; then
             flist=pyuno_list.txt
         else
             flist=mailmerge_list.txt
@@ -295,8 +299,8 @@ if test "z$OOO_VENDOR" != "zDebian" ; th
     fi
 
 
-    # Mandriva packaging
-    if test "$OOO_VENDOR" = "Mandriva"; then
+    # ROSA packaging
+    if test "$OOO_VENDOR" = "Mandriva" -o "$OOO_VENDOR" = "OpenMandriva" -o "$OOO_VENDOR" = "ROSA"; then
         # Not used
         remove_file common_list.txt $INSTALLDIR/share/gallery/htmltheme.orig
         remove_file common_list.txt $INSTALLDIR/share/dict/ooo/dictionary.lst
