diff -up libreoffice-7.5.0.0/configure.ac.3~ libreoffice-7.5.0.0/configure.ac
--- libreoffice-7.5.0.0/configure.ac.3~	2022-12-08 20:29:09.000000000 +0100
+++ libreoffice-7.5.0.0/configure.ac	2022-12-16 01:56:12.277991145 +0100
@@ -5651,7 +5651,6 @@ if test "$cross_compiling" = "yes"; then
         instsetoo_native/util/openoffice.lst.in \
         config_host/*.in \
         sysui/desktop/macosx/Info.plist.in \
-        .vscode/vs-code-template.code-workspace.in \
         solenv/lockfile/autoconf.h.in \
         ) \
     | (cd CONF-FOR-BUILD && tar xf -)
@@ -14764,7 +14763,7 @@ AC_CONFIG_FILES([config_host.mk
                  hardened_runtime.xcent
                  instsetoo_native/util/openoffice.lst
                  sysui/desktop/macosx/Info.plist
-                 vs-code.code-workspace.template:.vscode/vs-code-template.code-workspace.in])
+                 ])
 
 AC_CONFIG_HEADERS([config_host/config_buildconfig.h])
 AC_CONFIG_HEADERS([config_host/config_buildid.h])
