--- libreoffice-5.3.0/vcl/unx/kde4/KDESalGraphics.cxx.0051~	2016-11-28 16:16:20.418798674 +0100
+++ libreoffice-5.3.0/vcl/unx/kde4/KDESalGraphics.cxx	2016-11-28 16:19:41.782990584 +0100
@@ -137,8 +137,13 @@ namespace
         option->state |= state;
         option->rect = image->rect();
 
-        QPainter painter(image);
-        QApplication::style()->drawComplexControl(element, option, &painter);
+        // Hack: Don't display tiny useless scrollbars
+        // Impress is known to crash when adding effects (segfault is triggered by 15x18 scrollbar)
+        if (!((option->type == QStyleOption::SO_Slider) && ((option->rect.width() * option->rect.height()) < 300)))
+        {
+            QPainter painter(image);
+            QApplication::style()->drawComplexControl(element, option, &painter);
+        }
     }
 
     void lcl_drawFrame(QStyle::PrimitiveElement element, QImage* image, QStyle::State const & state)
