diff -up libreoffice-24.8.0.2/configure.ac.3~ libreoffice-24.8.0.2/configure.ac
--- libreoffice-24.8.0.2/configure.ac.3~	2024-07-26 15:49:08.000000000 +0200
+++ libreoffice-24.8.0.2/configure.ac	2024-08-06 01:26:24.318967849 +0200
@@ -5828,7 +5828,6 @@ if test "$cross_compiling" = "yes"; then
         sysui/desktop/macosx/Info.plist.in \
         sysui/desktop/macosx/hardened_runtime.xcent.in \
         sysui/desktop/macosx/lo.xcent.in \
-        .vscode/vs-code-template.code-workspace.in \
         solenv/lockfile/autoconf.h.in \
         ) \
     | (cd CONF-FOR-BUILD && tar xf -)
@@ -15508,10 +15507,7 @@ AC_CONFIG_FILES([
                  bin/odfvalidator.sh
                  bin/officeotron.sh
                  instsetoo_native/util/openoffice.lst
-                 sysui/desktop/macosx/Info.plist
-                 hardened_runtime.xcent:sysui/desktop/macosx/hardened_runtime.xcent.in
-                 lo.xcent:sysui/desktop/macosx/lo.xcent.in
-                 vs-code.code-workspace.template:.vscode/vs-code-template.code-workspace.in])
+                 ])
 # map unix-style mount dirs to windows directories: /mnt/c/foobar -> C:/foobar
 # easier to do it in a postprocessing command than to modify every single variable
 AC_CONFIG_FILES([config_host.mk], [
