diff -up libreoffice-24.8.0.2/vcl/source/app/salplug.cxx.7~ libreoffice-24.8.0.2/vcl/source/app/salplug.cxx
--- libreoffice-24.8.0.2/vcl/source/app/salplug.cxx.7~	2024-07-26 15:49:08.000000000 +0200
+++ libreoffice-24.8.0.2/vcl/source/app/salplug.cxx	2024-08-06 01:30:55.924275966 +0200
@@ -184,6 +184,15 @@ const char* const* autodetect_plugin_lis
 #if ENABLE_KF5
         "kf5",
 #endif
+#if ENABLE_QT5
+        "qt5",
+#endif
+#if ENABLE_KF6
+        "kf6",
+#endif
+#if ENABLE_QT6
+        "qt6",
+#endif
 #if ENABLE_GTK3_KDE5
         "gtk3_kde5",
 #endif
@@ -201,9 +210,15 @@ const char* const* autodetect_plugin_lis
 #if ENABLE_KF6
         "kf6",
 #endif
+#if ENABLE_QT6
+        "qt6",
+#endif
 #if ENABLE_KF5
         "kf5",
 #endif
+#if ENABLE_QT5
+        "qt5",
+#endif
 #if ENABLE_GTK3_KDE5
         "gtk3_kde5",
 #endif
@@ -218,6 +233,21 @@ const char* const* autodetect_plugin_lis
 
     static const char* const pStandardFallbackList[] =
     {
+#if ENABLE_QT6
+        "qt6",
+#endif
+#if ENABLE_KF6
+        "kf6",
+#endif
+#if ENABLE_QT5
+        "qt5",
+#endif
+#if ENABLE_KF5
+        "kf5",
+#endif
+#if ENABLE_GTK3_KDE5
+        "gtk3_kde5",
+#endif
 #if ENABLE_GTK3
         "gtk3",
 #endif
@@ -227,6 +257,32 @@ const char* const* autodetect_plugin_lis
         nullptr
     };
 
+    static const char* const pGtkFallbackList[] =
+    {
+#if ENABLE_GTK3
+        "gtk3",
+#endif
+#if ENABLE_GTK3_KDE5
+        "gtk3_kde5",
+#endif
+#if ENABLE_QT6
+        "qt6",
+#endif
+#if ENABLE_KF6
+        "kf6",
+#endif
+#if ENABLE_QT5
+        "qt5",
+#endif
+#if ENABLE_KF5
+        "kf5",
+#endif
+#if ENABLE_GEN
+        "gen",
+#endif
+        nullptr
+    };
+
 #if ENABLE_HEADLESS
     static const char* const pHeadlessFallbackList[] =
     {
@@ -248,10 +304,10 @@ const char* const* autodetect_plugin_lis
               desktop == DESKTOP_UNITY ||
               desktop == DESKTOP_XFCE  ||
               desktop == DESKTOP_MATE )
-        pList = pStandardFallbackList;
-    else if (desktop == DESKTOP_PLASMA5 || desktop == DESKTOP_LXQT)
+        pList = pGtkFallbackList;
+    else if (desktop == DESKTOP_PLASMA5)
         pList = pKDEFallbackList;
-    else if (desktop == DESKTOP_PLASMA6)
+    else if (desktop == DESKTOP_PLASMA6 || desktop == DESKTOP_LXQT)
         pList = pPlasma6FallbackList;
 
     return pList;
@@ -325,27 +381,27 @@ SalInstance *CreateSalInstance()
 #elif defined(MACOSX)
         "osx",
 #else // !_WIN32 && !MACOSX
-#if ENABLE_GTK3
-        "gtk3",
+#if ENABLE_KF6
+        "kf6",
+#endif
+#if ENABLE_QT6
+        "qt6",
 #endif
 #if ENABLE_KF5
         "kf5",
 #endif
+#if ENABLE_QT5
+        "qt5",
+#endif
 #if ENABLE_GTK3_KDE5
         "gtk3_kde5",
 #endif
+#if ENABLE_GTK3
+        "gtk3",
+#endif
 #if ENABLE_GEN
         "gen",
 #endif
-#if ENABLE_QT5
-        "qt5",
-#endif
-#if ENABLE_KF6
-        "kf6",
-#endif
-#if ENABLE_QT6
-        "qt6",
-#endif
 #endif // !_WIN32 && !MACOSX
         nullptr
     };
