--- libreoffice-4.1.0/sdext/source/pdfimport/xpdfwrapper/pdfioutdev_gpl.cxx.omv~	2013-08-30 00:46:37.497225750 +0200
+++ libreoffice-4.1.0/sdext/source/pdfimport/xpdfwrapper/pdfioutdev_gpl.cxx	2013-08-30 00:46:45.746069080 +0200
@@ -491,7 +491,7 @@ PDFOutDev::~PDFOutDev()
     delete m_pUtf8Map;
 }
 
-void PDFOutDev::startPage(int /*pageNum*/, GfxState* state)
+void PDFOutDev::startPage(int /*pageNum*/, GfxState* state, XRef *xref)
 {
     assert(state);
     printf("startPage %f %f\n",
--- libreoffice-4.1.0/sdext/source/pdfimport/xpdfwrapper/pdfioutdev_gpl.hxx.omv~	2013-08-30 00:45:27.080563188 +0200
+++ libreoffice-4.1.0/sdext/source/pdfimport/xpdfwrapper/pdfioutdev_gpl.hxx	2013-08-30 00:46:35.825257506 +0200
@@ -178,7 +178,7 @@ namespace pdfi
         virtual void setDefaultCTM(double *ctm) SAL_OVERRIDE;
 
         // Start a page.
-        virtual void startPage(int pageNum, GfxState *state) SAL_OVERRIDE;
+        virtual void startPage(int pageNum, GfxState *state, XRef *xref = 0) SAL_OVERRIDE;
 
         // End a page.
         virtual void endPage() SAL_OVERRIDE;
