diff -up libreoffice-7.1.0/configure.ac.3~ libreoffice-7.1.0/configure.ac
--- libreoffice-7.1.0/configure.ac.3~	2020-11-13 22:21:50.007939953 +0100
+++ libreoffice-7.1.0/configure.ac	2020-11-13 22:22:19.733325777 +0100
@@ -10800,7 +10800,7 @@ else
         AC_MSG_RESULT([external])
         SYSTEM_QRCODEGEN=TRUE
         AC_LANG_PUSH([C++])
-        AC_CHECK_HEADER(qrcodegen/QrCode.hpp, [],
+        AC_CHECK_HEADER(qrcodegencpp/QrCode.hpp, [],
             [AC_MSG_ERROR(qrcodegen headers not found.)], [#include <stdexcept>])
         AC_CHECK_LIB([qrcodegencpp], [main], [:],
             [ AC_MSG_ERROR(qrcodegen C++ library not found.) ], [])
diff -up libreoffice-7.1.0/cui/source/dialogs/QrCodeGenDialog.cxx.3~ libreoffice-7.1.0/cui/source/dialogs/QrCodeGenDialog.cxx
--- libreoffice-7.1.0/cui/source/dialogs/QrCodeGenDialog.cxx.3~	2020-11-13 22:21:50.007939953 +0100
+++ libreoffice-7.1.0/cui/source/dialogs/QrCodeGenDialog.cxx	2020-11-13 22:23:12.731014689 +0100
@@ -19,7 +19,7 @@
 
 #if ENABLE_QRCODEGEN
 #if defined(SYSTEM_QRCODEGEN)
-#include <qrcodegen/QrCode.hpp>
+#include <qrcodegencpp/QrCode.hpp>
 #else
 #include <QrCode.hpp>
 #endif
