diff -up libreoffice-7.1.0/configure.ac.3~ libreoffice-7.1.0/configure.ac
--- libreoffice-7.1.0/configure.ac.3~	2020-10-27 20:51:07.182118122 +0100
+++ libreoffice-7.1.0/configure.ac	2020-10-27 20:58:16.469701529 +0100
@@ -8102,6 +8102,7 @@ if test "$ENABLE_JAVA" != "" -a "$cross_
 
             # set to limit VM usage for JunitTests
             JAVAIFLAGS=-Xmx64M
+            JAVAIFLAGS="$JAVAIFLAGS -Djdk.net.URLClassPath.disableClassPathURLCheck=true"
             # set to limit VM usage for javac
             JAVACFLAGS=-J-Xmx128M
         fi
